modules/genomic_browser/php/filemanager.class.inc

```diff
@@ -55,10 +54,7 @@ class FileManager extends \NDB_Page implements ETagCalculator
      */
     private function _handleGET(ServerRequestInterface $request) : ResponseInterface
     {
-        $filter      = new HasAnyPermissionOrUserSiteMatch(
-            ['genomic_browser_view_allsites']
-        );
-        $provisioner = (new FilesProvisioner())->filter($filter);
```

Comment:
This code was removed because no site information is available from the query. HasAnyPermissionOrUserSiteMatch requires a getCenterID() returning the site ID.